core/src/Dscp/Resource/Logging.hs

```diff
@@ -56,22 +62,55 @@ getRealLoggerConfig LoggingParams{..} = do
             & lcTree .~ tree
     cfg <- readLoggerConfig lpConfigPath
     pure $ cfg <> cfgBuilder
+  where
+    readLoggerConfig :: MonadIO m => Maybe FilePath -> m LoggerConfig
+    readLoggerConfig = maybe (pure productionB) parseLoggerConfig
+
+getSyslogLoggerConfig :: MonadIO m => LoggingParams -> m (SyslogConfig, Maybe Text)
+getSyslogLoggerConfig LoggingParams{..} =
+    case lpConfigPath of
+        Nothing -> return (fallbackLogConfig, Just "No configuration file specified")
+        Just filePath -> syslogConfigFromFile filePath >>= \case
+            Right config -> return (config, Nothing)
+            Left e -> return (fallbackLogConfig, Just $ show e)
+  where
+    fallbackLogConfig = SyslogConfig [fallbackLoggerConfig]
+    fallbackLoggerConfig = defaultLoggerConfig
+        & loggerNameL .~ show lpDefaultName
+        & outOptionsL %~ (PERROR :)
+        & minLevelL %~ if lpDebug then const Debug else id
 
 allocLogging :: LoggingParams -> ComponentM LoggingIO
 allocLogging params = buildComponent "logging" pre fin
   where
-    pre = do
-        config <- getRealLoggerConfig params
-        (config', logging) <-
-            prepareLogWarper config (GivenName $ lpDefaultName params)
-        printCfg logging config
-        printCfg logging config'
-        return logging
-    printCfg logging finalConfig =
+    selectedType = fromMaybe Syslog $ lpLoggingType params -- Syslog is default
+
+    pre = case selectedType of
+        Warper -> do
+            config <- getWarperLoggerConfig params
+            (config', logging) <-
+                prepareLogWarper config (GivenName $ lpDefaultName params)
+            printCfg logging config Nothing
+            printCfg logging config' Nothing
```

Comment:
This will print the same message twice with slightly different configs. Why? I find this kind of misleading and redundant.

Reply:
I don't know why, I left it (for `log-warper` only) because it was there before, should I print just the second one?

Reply:
Ah, missed the fact it was here before. Yes, I think, printing the second one is sufficient.